groups/chips-rep-db/variables.tf

```diff
@@ -264,3 +264,12 @@ variable "netapp_ips" {
   type        = list(string)
   description = "List of Netapp IP addresses to use for iscsi discovery."
 }
+
+# ------------------------------------------------------------------------------
+# CHIPS DB SG variables
+# ------------------------------------------------------------------------------
+variable "chips_db_sg" {
+  type        = list(any)
+  description = "List of CHIPS DB SGs"
```

Comment:
Same as above

Reply:
Changed to description = "List of CHIPS DB REP Security Groups"